---
title: DMCM2018
layout: default
redirect_from: "/events/3rdCommunityMeeting/"
permalink: /DMCM2018
---

<img src="../images/places/Paris16.jpg" alt="Paris"/>

<p>  </p>

The 3rd Disease Maps Community Meeting (DMCM2018) is hosted by the <a href="https://sysbio.curie.fr/">Computational Systems Biology of Cancer group, Institut Curie</a>.

## Key dates

<b>Abstract submission deadline:</b> 15 May 2018<br />
<b>Decision on abstract selection:</b> 20 May 2018<br />
<b>Registration deadline:</b> 21 May 2018<br />
<b>DMCM2018:</b> 21-22 June 2018  

## Objectives

The objective of the 3rd DMCM is to consolidate efforts around the disease map projects and extend the Disease Maps Community. The 3rd DMCM will expose progress of the projects from the Disease Maps Community and beyond. The representative examples of network maps generation, their applications for disease data analysis, disease comorbidity, drug response prediction, advances in tools development and modelling approaches will be presented.

## Topics
Among others, the following topics will be discussed: biocuration of prior knowledge related to different human diseases; creation and maintenance of disease network maps following common standards; sharing sub-maps with mechanisms implemented into different human diseases; providing technical solutions for network maps complexity management, mathematical modelling of maps and web-tools for explorations in maps. <b>Topic-specific roundtable discussions</b> will be organised to identify bottleneck problems and ways to overcome them.  

## Hands-on tutorials

Part of the meeting will be dedicated to <b>demos and hands-on tutorials</b> on major tools supporting the disease maps.

## Programme

### 21 June

<table>
<tr><td><strong>09:00 - 09:15 Opening session and welcome</strong><br />
<a href="https://science.institut-curie.org/research/integrated-biology/bioinformatics-biostatistics-epidemiology-and-computational-systems-biology-of-cancer/team-barillot/">Emmanuel Barillot,</a> Institute Curie, Paris, France<br />
<a href="http://www.eisbm.org/portfolio/charles-auffray/">Charles Auffray,</a> European Institute for Systems Biology and Medicine, Lyon, France</td></tr>
<tr><td><strong>09:15 - 09:30 Aims and perspectives of the DMCM2018 meeting</strong><br />
<a href="http://www.eisbm.org/portfolio/alexander-mazein/">Alexander Mazein,</a> European Institute for Systems Biology and Medicine, Lyon, France<br/>
<a href="http://sysbio.curie.fr/people.html">Inna Kuperstein,</a> Institute Curie, Paris, France</td></tr> 
<tr><td><strong>09:30 - 09:45 A disease map for the blood pressure and glomerular filtration regulatory network on which antihypertensive and analgesic drugs act</strong><br />
<a href="http://fisiofarma.usal.es/index.php?option=com_content&view=article&id=205&Itemid=21">Francisco J. Lopez Hernandez,</a> University of Salamanca, Salamanca, Spain</td></tr>
<tr><td><strong>09:45 - 10:00 Semi-automatic generation of disease maps: focus on asthma</strong><br />
<a href="http://www.eisbm.org/portfolio/alexander-mazein/">Alexander Mazein,</a> European Institute for Systems Biology and Medicine, Lyon, France</td></tr>
</table>

<p><i>10:00 - 10:30 Coffee break</i></p>  

<table>
<tr><td><strong>10:30 - 10:45 A multiscale signalling network map of immune response in cancer reveals signatures of cell heterogeneity and functional polarization</strong><br />
<a href="https://sysbio.curie.fr/people.html">Maria Kondratova,</a> Institute Curie, Paris, France<br />
<a href="/pages/events/DMCM2018_abstracts/Maria_Kondratova_etal_DMCM2018.pdf"><b><font color="Navy"><span style="text-decoration: underline;">Abstract</span></font></b></a></td></tr>
<tr><td><strong>10:45 - 11:30 Talks selected from abstracts</strong></td></tr> 
<tr><td><strong>11:30 - 12:30 Invited talk: WikiPathways: curation, visualization and analysis of biological pathways</strong><br />
<a href="https://www.maastrichtuniversity.nl/martina.kutmon">Martina-Summer Kutmon</a> Maastricht University, Maastricht, the Netherlands<br />
<a href ="/pages/events/DMCM2018_abstracts/Invited_Talk_Martina_Kutmon_etal_DMCM2018.pdf"><b><font color="Navy"><span style="text-decoration: underline;">Abstract</span></font></b></a></td></tr>
</table>
<p><i>12:30 - 13:30 Lunch</i></p>  

<p><strong>13:30 - 14:15 Hands-on tutorials: First round</strong><br/>
<strong>14:15 - 15:00 Hands-on tutorials: Second round</strong></p>

<table>
<tr><td><strong>Tutorial 1: MINERVA for visualization of disease maps</strong><br />
<a href="https://wwwen.uni.lu/lcsb/people/piotr_gawron">Piotr Gawron,</a> 
<a href="https://wwwfr.uni.lu/lcsb/people/marek_ostaszewski">Marek Ostaszewski,</a> Luxembourg Centre for Systems Biomedicine, Belval, Luxembourg<br />
<a href="/pages/events/DMCM2018_abstracts/Tutorial_MINERVA_Piotr_Gawron_etal_DMCM2018.pdf"><b><font color="Navy"><span style="text-decoration: underline;">Abstract</span></font></b></a></td></tr>
<tr><td><strong>Tutorial 2: Constructing and visualizing pathways with web-based SBGN editor Newt</strong><br />
<a href="http://www.cs.bilkent.edu.tr/~ugur/">Ugur Dogrusoz,</a> Bilkent University, Ankara, Turkey<br />
<a href="/pages/events/DMCM2018_abstracts/Tutorial_Newt_Ugur_Dugrosoz_DMCM2018.pdf"><b><font color="Navy"><span style="text-decoration: underline;">Abstract</span></font></b></a></td></tr> 
<tr><td><strong>Tutorial 3: COBRA toolbox for visualisation and map manipulation through CellDesigner a nd MatLab environment</strong><br />
<a href="https://wwwen.uni.lu/lcsb/people/jennifer_modamio_chamarro">Jennifer Modamio,</a><a href="https://wwwfr.uni.lu/lcsb/people/ronan_mt_fleming"> Ronan Fleming,</a> Luxembourg Centre for Systems Biomedicine, Belval, Luxembourg<br />
<a href="http://sysbio.curie.fr/people.html">Nicolas Sompairac,</a><a href="http://sysbio.curie.fr/people.html"> Andrei Zinovyev,</a><a href="http://sysbio.curie.fr/people.html"> Inna Kuperstein,</a> Institute Curie, Paris, France<br />
<a href="/pages/events/DMCM2018_abstracts/Tutorial_COBRA_Jennifer_Modamio_etal_DMCM2018.pdf"><b><font color="Navy"><span style="text-decoration: underline;">Abstract</span></font></b></a></td></tr>
<tr><td><strong>Tutorial 4: rROMA, a tool for module activity calculation from omics data and networks</strong><br />
<a href="https://sysbio.curie.fr/people.html">Luca Albergante,</a><a href="https://sysbio.curie.fr/people.html"> Urszula Czerwinska,</a><a href="https://sysbio.curie.fr/people.html"> Andrei Zinovyev,</a><a href="https://sysbio.curie.fr/people.html"> Loredana Martignetti,</a> Institute Curie, Paris, France<br />
<a href="/pages/events/DMCM2018_abstracts/Tutorial_rROMA_Luca_Albergante_etal_DMCM2018.pdf"><b><font color="Navy"><span style="text-decoration: underline;">Abstract</span></font></b></a></td></tr>
<tr><td><strong>Tutorial 5: From interaction maps to dynamical models with Cell Collective: A hands on tutorial</strong><br />
<a href="https://biochem.unl.edu/tomas-helikar">Tomas Helikar,</a> University of Nebraska-Lincoln, Lincoln, United States<br />
<a href="/pages/events/DMCM2018_abstracts/Tutorial_Models_Tomas_Helikar_DMCM2018.pdf"><b><font color="Navy"><span style="text-decoration: underline;">Abstract</span></font></b></a></td></tr>
<tr><td><strong>Tutorial 6: mEPN and yEd: a Graphical and Computational Modelling Platform for Biological Pathways</strong><br />
<a href="https://www.ed.ac.uk/roslin/about/contact-us/staff/tom-freeman">Tom C. Freeman,</a> University of Edinburgh, Edinburgh, UK<br />
<a href="/pages/events/DMCM2018_abstracts/Tutorial_mEPN_Tom_Freeman_DMCM2018.pdf"><b><font color="Navy"><span style="text-decoration: underline;">Abstract</span></font></b></a></td></tr>
</table>

<p><i>15:00 - 15:30 Coffee break</i></p>  

<table>
<tr><td><strong>15:30 - 15:45 Metabolic and signalling network map integration: application to cross-talk studies and omics data analysis in cancer</strong><br />
<a href="https://sysbio.curie.fr/people.html">Nicolas Sompairac,</a> Institute Curie, Paris, France<br />
<a href="/pages/events/DMCM2018_abstracts/Nicolas_Sompairac_DMCM2018.pdf"><b><font color="Navy"><span style="text-decoration: underline;">Abstract</span></font></b></a></td></tr> 
<tr><td><strong>15:45 - 16:00 MINERVA API and plugin architechture: new data visualization interfaces for disease maps</strong><br />
<a href="https://wwwen.uni.lu/lcsb/people/piotr_gawron">Piotr Gawron,</a> Luxembourg Centre for Systems Biomedicine, Belval, Luxembourg<br />
<a href="/pages/events/DMCM2018_abstracts/Marek_Ostaszewski_etal_DMCM2018.pdf"><b><font color="Navy"><span style="text-decoration: underline;">Abstract</span></font></b></a></td></tr> 
<tr><td><strong>16:00 - 17:15 Title TBA</strong><br />
<a href="https://www.lih.lu/page/departments/dii-all5-immune-systems-biology-1384">Feng He,</a> Luxembourg Institute of Health, Luxembourg, Luxembourg</td></tr>
<tr><td><strong>17:15 - 17:30 Integrating disease maps using a graph database approach</strong><br />
<a href="http://www.eisbm.org/portfolio/irina-balaur/">Irina Balaur,</a> European Institute for Systems Biology and Medicine, Lyon, France<br />
<a href="/pages/events/DMCM2018_abstracts/Irina_Balaur_etal_DMCM2018.pdf"><b><font color="Navy"><span style="text-decoration: underline;">Abstract</span></font></b></a></td></tr> 
<tr><td><strong>17:30 - 18:00 Talks selected from abstracts</strong></td></tr>
<tr><td><strong>18:00 - 19:00 Project managers' discussion</strong></td></tr>
</table>
<p><i>19:00 Dinner</i></p> 

### 22 June

<table>
<tr><td><strong>09:00 - 09:15 Parkinson's disease map facilitated gene expression analysis reveals new insights in PD pathogenesis</strong><br />
<a href="https://wwwen.uni.lu/lcsb/people/stephan_gebel">Stephan Gebel,</a> Luxembourg Centre for Systems Biomedicine, Belval, Luxembourg<br />
<a href="/pages/events/DMCM2018_abstracts/Stephan_Gebel_etal_DMCM2018.pdf"><b><font color="Navy"><span style="text-decoration: underline;">Abstract</span></font></b></a></td></tr> 
<tr><td><strong>09:15 - 09:30 Application of disease maps to rare diseases: as muscular dystrophies</strong><br />
<a href="https://www.sjdhospitalbarcelona.org/es/cecilia-jimenez-mallebrera">Cecilia Jimenez Mallebrera,</a> Hospital Sant Joan de Deu, Barcelona, Spain<br />
<a href="/pages/events/DMCM2018_abstracts/Cecilia_Jimenez_etal_DMCM2018.pdf"><b><font color="Navy"><span style="text-decoration: underline;">Abstract</span></font></b></a></td></tr> 
<tr><td><strong>09:30 - 09:45 Comprehensive signaling network of regulated cell death: comparison of cell death modes in Alzheimer's neurodegenerative disease and cancer</strong><br />
<a href="https://sysbio.curie.fr/people.html">Cristobal Monraz,</a> Institute Curie, Paris, France<br />
<a href="/pages/events/DMCM2018_abstracts/Cristobal_Monraz_etal_DMCM2018.pdf"><b><font color="Navy"><span style="text-decoration: underline;">Abstract</span></font></b></a></td></tr> 
<tr><td><strong>09:45 - 10:00 Towards cystic fibrosis disease map modelling</strong><br />
<a href="http://lasige.di.fc.ul.pt/user/508">Catarina Pereira</a> and <a href="http://www.di.fc.ul.pt/~afalcao/">Andre Falcao,</a> University of Lisbon, Lisbon, Portugal</td></tr>
</table>

<p><i>10:00 - 10:30 Coffee break</i></p> 

<table>
<tr><td><strong>10:30 - 10:45 Community Resources Connecting Chemistry and Toxicity Knowledge to Environmental Observations</strong><br />
<a href="https://wwwen.uni.lu/lcsb/people/emma_schymanski">Emma Schymanski,</a> Luxembourg Centre for Systems Biomedicine, Belval, Luxembourg<br>
<a href="http://epa.academia.edu/AntonyWilliams"> Antony Williams,</a> National Center for Computational Toxicology, Office of Research and Development, U.S. Environmental Protection Agency, Durham,USA<br />
<a href="/pages/events/DMCM2018_abstracts/Emma_Schymanski_etal_DMCM2018.pdf"><b><font color="Navy"><span style="text-decoration: underline;">Abstract</span></font></b></a></td></tr> 
<tr><td><strong>10:45 - 11:30 Talks from selected abstracts</strong></td></tr>
<tr><td><strong>11:30 - 12:30 Invited talk: Reactome multi-scale pathway visualization</strong><br />
<a href="https://www.ebi.ac.uk/about/people/henning-hermjakob">Henning Hermjakob,</a> Molecular Systems services EMBL-EBI, Cambridge, United Kingdom<br />
<a href="/pages/events/DMCM2018_abstracts/Invited_Talk_Henning_Hermjacob_DMCM2018.pdf"><b><font color="Navy"><span style="text-decoration: underline;">Abstract</span></font></b></a></td></tr>
</table>

<p><i>12:30 - 13:30 Lunch</i></p>  

<table>
<tr><td><strong>13:30 - 13:45 Computational systems biology approach for the study of rheumatoid arthritis: from a molecular map to a dynamical model</strong><br />
<a href="https://www.researchgate.net/profile/Anna_Niarakis">Anna Niarakis,</a> Universite d' Evry Val d' Essonne, Evry Val d' Essonne, France<br />
<a href="/pages/events/DMCM2018_abstracts/Anna_Niarakis_DMCM2018.pdf"><b><font color="Navy"><span style="text-decoration: underline;">Abstract</span></font></b></a></td></tr> 
<tr><td><strong>13:45 - 14:00 Status report: parameter estimation of a large-scale mechanistic model for mast cells in asthma</strong><br />
<a href="https://www.softmatter.physik.uni-muenchen.de/personen/guests/ligon/index.html">Thomas S. Ligon,</a> Ludwig-Maximilians-Universitat, Munich, Germany<br><a href="https://www.helmholtz-muenchen.de/icb/research/groups/data-driven-computational-modelling/group-members/staff/ma/2519/Dr.-Ing.-Hasenauer/index.html"> Jan Hasenauer,</a> Helmholtz Zentrum Munchen, Munich, Germany<br />
<a href="/pages/events/DMCM2018_abstracts/Thomas_Ligon_etal_DMCM2018.pdf"><b><font color="Navy"><span style="text-decoration: underline;">Abstract</span></font></b></a></td></tr> 
<tr><td><strong>14:00 - 14:15 SBGN-ML, SBML visual packages, CellDesigner. Which? Why? How?</strong><br />
<a href="https://www.babraham.ac.uk/our-research/signalling/nicolas-le-novere">Nicolas Le Novere,</a> Babraham Institute, Babraham, United Kingdom<br />
<a href="/pages/events/DMCM2018_abstracts/Nicolas_LeNovere_DMCM2018.pdf"><b><font color="Navy"><span style="text-decoration: underline;">Abstract</span></font></b></a></td></tr> 
<tr><td><strong>14:15 - 15:15 Invited talk: Share FAIR – Data Management and Standards for Systems Medicine</strong><br />
<a href="https://www.h-its.org/en/research/sdbv/people/">Martin Golebiewski,</a> Heidelberg Institute for Theoretical Studies, Heidelberg, Germany</td></tr>
</table>

<p><i>15:15 - 15:45 Coffee break</i></p>

<table>
<tr><td><strong>15:45 - 17:15 Thematic discussion groups (suggested topics):</strong><br />
Integration of maps in a shared repository<br />
Tools for maps generation<br />
Applications of disease maps in research and clinics: what is missing to close the gap?<br />
Maps to models: curation standards to allow easy transformation of networks into executable models<br />
Maps' complexity management approaches<br />
Disease pathways vs. healthy pathways: how different are they?</td></tr>
<tr><td><strong>17:15 - 17:30 Conclusions and closing session</strong><br />
<a href="https://science.institut-curie.org/research/integrated-biology/bioinformatics-biostatistics-epidemiology-and-computational-systems-biology-of-cancer/team-barillot/">Emmanuel Barillot,</a> Institute Curie, Paris, France<br />
<a href="http://www.eisbm.org/portfolio/charles-auffray/">Charles Auffray,</a> European Institute for Systems Biology and Medicine, Lyon, France</td></tr>
</table>

## Abstract submission

Submit abstracts (up to 400 words) via <a href="https://easychair.org/conferences/?conf=dmcm2018"><strong><font color="Navy">the submission link</font></strong></a> for oral presentations and/or posters. Indicate 4 keywords to allocate it into the suitable session.  

## Registration

The registration is free but mandatory. Fill in the <a href="https://goo.gl/forms/7M8aG1hF56yakaRx2"><strong><font color="Navy">registration form here</font></strong></a>.  

## Location

<b>Batiment de biologie du developpement et cancer (BDD building)</b><br />
11 rue Pierre et Marie Curie<br />
75005, Paris<br />
France

<iframe src="https://www.google.com/maps/embed?pb=!1m14!1m8!1m3!1d1312.8673854714116!2d2.3414244!3d48.8441976!3m2!1i1024!2i768!4f13.1!3m3!1m2!1s0x47e671c29b2f9877%3A0x7a230a3bff909a90!2sInstitut+Curie-p%C3%B4le+Biologie+du+D%C3%A9veloppement!5e0!3m2!1ses-419!2sfr!4v1518696183461" width="480" height="360" frameborder="0" style="border:0" allowfullscreen></iframe>

<br />

## Travel

Coming soon

## Accommodation

<p>Recommended hotels:</p>
<p><a href="https://www.les-jardins-du-luxembourg.com/fr/">Les Jardins Du Luxembourg</a>
<br />5 Impasse Royer-Collard, 75005 Paris, France</p>
<p><a href="http://www.elysa-hotel-paris.com/">Hotel Elysa-Luxembourg</a>
<br />6 Rue Gay Lussac, 75005 Paris, France</p>
<p><a href="https://bresil-paris-hotel.com/fr/">Hotel du Brésil</a>
<br />10 rue Le Goff, 75005 Paris, France</p>
<p><a href="http://www.hotelmicheletodeon.com/fr">Hotel Michelet Odeon</a>
<br />6 Place De L Odeon, 75006 Paris, France</p>
<p><a href="https://hoteldelavenir.com/fr/">Hotel de l'Avenir</a>
<br />65 rue Madame, 75006 Paris, France</p>
<p><a href="http://www.hotel-desmines-paris.com/fr/">Hôtel Des Mines</a>
<br />125 Bd Saint Michel, 75005 Paris, France</p>

## Contact

<p><a href="mailto:disease-maps-coord@googlegroups.com ">disease-maps-coord@googlegroups.com</a></p>

## Organisers

<p><a href="https://sysbio.curie.fr/">Computational Systems Biology of Cancer group, Institut Curie</a>:
<br /><a href="mailto:emmanuel.barillot@curie.fr ">Emmanuel Barillot</a>
<br /><a href="mailto:maria.kondratova@curie.fr ">Maria Kondratova</a>
<br /><a href="mailto:inna.kuperstein@curie.fr ">Inna Kuperstein</a>
<br /><a href="mailto:luis-cristobal.monraz@curie.fr ">Cristobal Monraz</a> 
<br /><a href="mailto:andrei.zinovyev@curie.fr ">Andrei Zinovyev</a>
</p>

## Disease Maps Project Coordinators

<p><a href="mailto:amazein@eisbm.org">Alexander Mazein</a>
<br /><a href="mailto:marek.ostaszewski@uni.lu">Marek Ostaszewski</a>
<br /><a href="mailto:inna.kuperstein@curie.fr ">Inna Kuperstein</a>
<br /><a href="mailto:andrei.zinovyev@curie.fr ">Andrei Zinovyev</a>
<br /><a href="mailto:reinhard.schneider@uni.lu">Reinhard Schneider</a>
<br /><a href="mailto:emmanuel.barillot@curie.fr ">Emmanuel Barillot</a>
<br /><a href="mailto:rudi.balling@uni.lu">Rudi Balling</a>
<br /><a href="mailto:cauffray@eisbm.org">Charles Auffray</a>
</p>
