---
title: DMCM2020
layout: default
permalink: /DMCM2020
---



<img src="../images/places/covid-19-banner.jpg"/>

The key theme of the 2020 Disease Maps Community Meeting is <a href="https://covid.pages.uni.lu/map_curation">COVID-19</a>. The event is organised as a virtual conference by the <a target="_blank" href="https://wwwen.uni.lu/lcsb">Luxembourg Centre for Systems Biomedicine</a>.

## Key dates

Abstract submission deadline: <strong><font color="red">EXTENDED until 31 October 2020</font></strong>  
Acceptance notification: **30 October 2020**  
Final programme: **2 November 2020**  
Registration deadline: **5 November 2020**  
DMCM2020: **12-14 November 2020**  

## Abstract submission

Abstract submission is now closed.

<!--Please submit your abstract up to 400 words via this <a href="https://easychair.org/conferences/?conf=dmcm2020"><strong><font color="Navy">submission link</font></strong></a> by <font color="red">31 October 2020 (extended)</font>.   -->

<!--<p style="color:red;">Abstract submission deadline: <strong><font color="red">31 October 2020 (extended)</font></strong>.</p>-->


## Registration

The registration is free but mandatory. Please fill in the <a href="https://forms.gle/6wjFaQ68fCptdvQW6"><strong><font color="Navy">registration form here</font></strong></a> by <font color="red">5 November 2020</font>.  

<!--Registration deadline: **5 November 2020**.
<p style="color:red;">Registration deadline: <strong><font color="red">5 November 2020</font></strong>.</p>-->

## Virtual environment

This webpage is the main entry point for accessing all events of the programme.  We run this meeting using a combination of the virtual platform ZENODO and Webex Meetings. The live talks and the discussion panels will take place in Webex rooms, whereas the personal profiles of the participants and virtual posters (PDFs plus 7 minutes pre-recorded videos) will be displayed on the ZENODO platform.  Upon agreement with the speakers, the live talks will be recorded and also displayed on the ZENODO platform right after the session.  

## How to?

The detailed instructions about profile generation, poster standards, video recordings, materials unloading to the ZENODO platform, etc. will be provided right after the abstract notification announcement.  
 
## If I am just an attendee?

We encourage everyone to submit the abstract and present the work. However, attendance without project presentation is also welcome. The attendees need to register to the meeting by 5 November 2020. The explanations on the virtual modalities of the presentations, discussion and posters will be available on the meeting website.  
 
## Topics

The key theme of the 2020 Disease Maps Community Meeting is COVID-19. The first day of the meeting will be dedicated to COVID-19, the second and the third days will be dedicated to ongoing community projects.  
 
We also welcome talks, posters and discussions on other topics relevant to disease maps: ongoing disease map projects, related technologies and disease map applications.  
 
The programme will contain keynotes, invited talks and talks selected from abstracts. We’ll hold several parallel discussion panels on the major burning topics in the field. All virtual posters will be exposed on the ZENODO platform. 

## Invited speakers

<table>
<tr>
<td style="width: 220px;"><p style="margin:4px;"><img src="/images/teamhq/VassiliSoumelis.jpg" width="150"/></p></td>
<td style="width: 220px;"><p style="margin:4px;"><img src="/images/teamhq/OlivierLantz.jpg" width="150"/></p></td>
<td style="width: 220px;"><p style="margin:4px;"><img src="/images/teamhq/ChrisSander.jpg" width="150"/></p></td>
</tr>
<tr>
<td style="width: 220px;"><p style="margin:4px;"><strong>Vassili Soumelis, MD, PhD</strong></p><p style="margin:4px; line-height:100%;"><font size="2">Hôpital St Louis, Paris, France</font></p></td>
<td style="width: 220px;"><p style="margin:4px;"><strong>Olivier Lantz, MD, PhD</strong></p><p style="margin:4px; line-height:100%;"><font size="2">Institut Curie, Paris, France</font></p><br /></td>
<td style="width: 220px;"><p style="margin:4px;"><strong>Chris Sander, PhD</strong></p><p style="margin:4px; line-height:100%;"><font size="2">Dana–Farber Cancer Institute, Harvard Medical School, Boston, US</font></p><br /></td>
</tr>
<tr>
<td style="width: 220px;"><p style="margin:4px;"><img src="/images/teamhq/MariaMartin.jpg" width="150"/></p></td>
<td style="width: 220px;"><p style="margin:4px;"><img src="/images/teamhq/DagmarWaltemath.jpg" width="150"/></p></td>
<td style="width: 220px;"><p style="margin:4px;"><img src="/images/teamhq/InaKoch.jpg" width="150"/></p></td>
</tr>
<tr>
<td style="width: 220px;"><p style="margin:4px;"><strong>Maria-Jesus Martin, PhD</strong></p><p style="margin:4px; line-height:100%;"><font size="2"><!--European Molecular Biology Laboratory, European Bioinformatics Institute-->EMBL-EBI, Cambridge, UK</font></p></td>
<td style="width: 220px;"><p style="margin:4px;"><strong>Dagmar Waltemath, PhD</strong></p><p style="margin:4px; line-height:100%;"><font size="2">Greifswald University Hospital, Greifswald, Germany</font></p></td>
<td style="width: 220px;"><p style="margin:4px;"><strong>Ina Koch, PhD</strong></p><p style="margin:4px; line-height:100%;"><font size="2">Goethe University Frankfurt, Frankfurt, Germany</font></p></td>
</tr>
</table>

<!--## Confirmed talks
| **Single cell RNAseq analysis of severe COVID-19 patients antigen-presenting cells**<br />_[Vassili Soumelis](https://science.curie.fr/members/vassili-soumelis-md-phd/), Hôpital St Louis, Paris, France_ |
| **The Curiosa study: high seroprevalence but short-lived immune response to SARS-CoV-2 infection in Paris**<br />_[Olivier Lantz](https://institut-curie.org/personne/olivier-lantz), Institut Curie, Paris, France_ |
| **Informatics resources for COVID-19 proteomics, viral evolution and drug effects** <br />_[Chris Sander](https://www.dfhcc.harvard.edu/insider/member-detail/member/chris-sander-phd/), Dana–Farber Cancer Institute, Harvard Medical School, Boston, US_ | 
| **Deciphering molecular mechanisms of disease: a view on protein knowledge and disease maps**<br />_[Maria-Jesus Martin](https://www.ebi.ac.uk/about/people/maria-jesus-martin), European Molecular Biology Laboratory, European Bioinformatics Institute (EMBL-EBI), Cambridge, UK_ |
| **Adding value to scientific results using COMBINE standards and guidelines for modelling and simulation in systems medicine**<br />_[Dagmar Waltemath](https://www.fis.med.uni-greifswald.de/FIS/init_person_browser.action?pers_id=ngpocpv7uc2ss), Greifswald University Hospital, Greifswald, Germany_ |
| **Petri nets for the analysis of disease maps: applied to the canonical and non-canonical NF-κB pathways**<br />_[Ina Koch](https://www.uni-frankfurt.de/57211826/People), Goethe University Frankfurt, Frankfurt, Germany_ |-->


## Programme

The time is shown for Paris: Central European Time (CET) &ndash; **UTC/GMT +1 hour**.  
Please check your local time [here](https://www.thetimezoneconverter.com/?t=09:00&tz=Paris&).

### <strong><font color="Blue">Thursday 12 November</font></strong>

### <strong><font color="Blue">Session 1</font></strong> ([Meeting room A](https://unilu.webex.com/unilu/j.php?MTID=mf57af1440a92c6ed3df9eb8820f63e50))

**09.00-09.30 [Talk 1]:**
**Introductory lecture: COVID-19 Disease Map project**
*Marek Ostaszewski*

**09.30-10.10 [Talk 2]:**
<strong><font color="red">KEYNOTE</font></strong> **The Curiosa study: high seroprevalence but short-lived immune response to SARS-CoV-2 infection in Paris**
*Olivier Lantz*

**10.10-10.30 [Talk 3]:**
**Connecting the COVID-19 dots: susceptibility factors and their immunodynamics in SARS-CoV-2 infection and disease severity**
*Rupsha Fraser, Aurelio Orta-Resendiz and Alexander Mazein*

**10.30-10.50 [Talk 4]:**
**Looking for pathways related to COVID-19 phenotypes: Confirmation of pathogenic mechanisms by SARS-CoV-2 - Host interactome**
*Francesco Messina, Emanuala Giombini, Chiara Montaldo, Mauro Piacentini, Maria R. Capobianchi, Francesco Nicola Lauria and Giuseppe Ippolito*

*10.50-11.10 Break and Posters session 1*

**11.10-11.30 [Talk 5]:**
**The WikiPathways COVID-19 Community Portal**
*Martina Kutmon, Friederike Ehrhart, Denise Slenter, Kristina Hanspers, Egon Willighagen, Alexander Pico and Chris Evelo*

**11.30-11.50 [Talk 6]:**
**ViralLink: An integrated workflow to investigate the effect of SARS-CoV-2 on intracellular signalling and regulatory pathways**
*Agatha Treveil, Dezso Modos and Tamas Korcsmaros*

**11.50-12.10 [Talk 7]:**
**Cov-Hipathia: A tool for mechanistic modeling of the SARS-CoV-2 disease map**
*Kinza Rian, Marina Esteban Medina, Devrim Gunyel, Carlos Loucera, Marek Ostaszewski, Anna Niarakis, Maria Peña-Chilet and Joaquin Dopazo*

*12.10-13.30 Lunch and Posters session 2*

### <strong><font color="Blue">Session 2</font></strong> ([Meeting room A](https://unilu.webex.com/unilu/j.php?MTID=mf57af1440a92c6ed3df9eb8820f63e50))

**13.30-14.00 [Talk 8]:**
**Introductory lecture: Modelling approaches in COVID-19 Disease Map project**
*Anna Niarakis*

**14.00-14.40 [Talk 9]:**
<strong><font color="red">KEYNOTE</font></strong> **Informatics resources for COVID-19 proteomics, viral evolution and drug effects**
*Chris Sander*

**14.40-15.00 [Talk 10]:**
**Reactome COVID-19: the literature curation strategy**
*Andrea Senff-Ribeiro, Robin Haw, Lisa Matthews, Solomon Shorser, Peter D'Eustachio and Marc Gillespie*

**15.00-15.20 [Talk 11]:**
**The INDRA Database and Network Search: Comprehensive, context-specific information on biochemical interactions**
*John Bachman, Benjamin Gyori, Patrick Greene and Klas Karis*

15.20-15.40 Break and Posters session 3

**15.20-15.40 [Talk 12]:**
**Coronavirus Interactome: The IMEx dataset of host-viral biomolecular interactions**
*Kalpana Panneerselvam, Pablo Porras Millan, Noemí del Toro, Margaret Duesbury, Livia Perfetto, Anjali Shrivastava, Sandra Orchard and Henning Hermjakob*

**16.00-16.20 [Talk 13]:**
**SEEK and Find: FAIRDOM Data Management Support for COVID-19 Disease Maps**
*Martin Golebiewski, Xiaoming Hu, Stuart Owen, Olga Krebs, Munazah Andrabi, Finn Bacall, Ina Poehner, Maja Rey, Jacky Snoep, Alan Williams, Ulrike Wittig, Katy Wolstencroft, Carole Goble and Wolfgang Müller*

**16.20-17.00 [Talk 14]:**
<strong><font color="red">KEYNOTE</font></strong> **Single cell RNAseq analysis of severe COVID-19 patients antigen-presenting cells**
*Vassili Soumelis*

**17.00-18.00 ([Meeting room A](https://unilu.webex.com/unilu/j.php?MTID=mf57af1440a92c6ed3df9eb8820f63e50) / Meeting room B) Virtual apéro**

### <strong><font color="Blue">Thursday 13 November</font></strong>

### <strong><font color="Blue">Session 3</font></strong> ([Meeting room A](https://unilu.webex.com/unilu/j.php?MTID=mf57af1440a92c6ed3df9eb8820f63e50))

**09.00-09.20 [Talk 15]:**
**Opening lecture**
*Inna Kuperstein*

**09.20-10.00 [Talk 16]:**
<strong><font color="red">KEYNOTE</font></strong> **Deciphering molecular mechanisms of disease: a view on protein knowledge and disease maps**
*Maria-Jesus Martin*

**10.00-10.20 [Talk 17]:**
**Discovering the role of intercellular communication and its downstream effect in healthy and diseased conditions**
*Lejla Gul, Dezso Modos, Denes Turei, Alberto Valdeolivas, Matthew Madgwick, Julio Saez-Rodriguez and Tamas Korcsmaros*

*10.20-10.40 Break and Posters session 4*

**10.40-11.00 [Talk 18]:**
**A multi-layer scalable representation of disease mechanisms: lessons learned from the AsthmaMap project**
*Alexander Mazein and Charles Auffray*	
 
**11.00-11.20 [Talk 19]:**
**Comprehensive Map of the Regulated Cell Death Signaling Network: A Powerful Analytical Tool for Studying Diseases**
*Jean-Marie Ravel, L. Cristobal Monraz Gomez, Nicolas Sompairac, Boris Zhivotovsky, Guido Kroemer, Emmanuel Barillot, Andrei Zinovyev and Inna Kuperstein*

**11.20-11.40 [Talk 20]:**
**Inference of a modular, large-scale Boolean network for modelling the Rheumatoid Arthritis fibroblast-like synoviocytes**
*Vidisha Singh, Sylvain Soliman and Anna Niarakis*
 
**11.40-12.00 [Talk 21]:**
**HCM Map: Interactive Knowledge Resource About Molecular Mechanisms of Hypertrophic Cardiomyopathy**
*Mila Glavaški and Lazar Velicki*

*12.00-13.30 Lunch and Posters session 5*

### <strong><font color="Blue">Session 4</font></strong> ([Meeting room A](https://unilu.webex.com/unilu/j.php?MTID=mf57af1440a92c6ed3df9eb8820f63e50))

**09.00-09.20 [Talk 22]:** **Opening lecture** *Alexander Mazein*

**13.50-14.10 [Talk 23]:**
**CyF-MAP: a pathway-based resource for Cystic Fibrosis**
*Catarina Pereira, Alexander Mazein, Margarida Amaral and Andre Falcao*

**14.10-14.30 [Talk 24]:**
**SysCID map: a computational knowledge repository for rheumatoid arthritis, systemic lupus erythematosus and inflammatory bowel disease**
*Marcio Acencio, Marek Ostaszewski, Wei Gu, Venkata Satagopam and Reinhard Schneider*

**14.30-14.50 [Talk 25]:**
**Towards modularity and composability for assembling large-scale process description maps**
*Adrien Rougny, Alexander Mazein, Joanthan Karr, Julio Saez Rodriguez, Marek Ostaszewski and Reinhard Schneider*

*14.50-15.10 Break and Posters session 6*

**15.10-15.30 [Talk 26]:**
<strong><font color="red">KEYNOTE</font></strong> **Adding value to scientific results using COMBINE standards and guidelines for modelling and simulation in systems medicine**
*Dagmar Waltemath*

**15.30-16.10 [Talk 27]:**
<strong><font color="red">KEYNOTE</font></strong> **Petri nets for the analysis of disease maps: applied to the TNFR1 signaling pathway**
*Ina Koch* 

*16.10-16.30 Break*

**16.30-16.50 [Talk 28]:**
**Newt: view, design and analyze pathways in SBGN and more**
*Ugur Dogrusoz*

**16.50-17.10 [Talk 29]:**
**Reactome DisGeNET overlay of gene-disease associations**
*Chuan Deng, Chuqiao Gong, Guilherme Viteri and Henning Hermjakob*

**17.10-17.30 [Talk 30]:**
**A suite of plugins for data-driven analyses of acute inflammatory phenotypes**
*Matti Hoch, Suchi Smita, Olaf Wolkenhauer and Shailendra Gupta*

**17.30-18.30 ([Meeting room A](https://unilu.webex.com/unilu/j.php?MTID=mf57af1440a92c6ed3df9eb8820f63e50) / Meeting room B) Virtual apéro**

### <strong><font color="Blue">Thursday 14 November</font></strong>

**10:00 - 10:30 [Talk 31]:**
**Introductory talk: COVID-19 Disease Map at the EUvsVirus Hackathon - making the map work** *Anna Niarakis*

**10:30 - 12:00 Breakout discussions**  

Proposed topics so far:  

1. Interoperability, common standards and practices for working together at a larger scale - lessons learned from the C19DM effort.
2. Disease Maps Community: the status, the past year and the roadmap.
3. How to do curation and integrate efforts of many groups: training materials, guidelines ([an example](https://doi.org/10.1093/database/bay036)).

Please suggest topics for breakout discussions. With suggestions pleaase write to [dmcm-organisers@googlegroups.com](mailto:dmcm-organisers@googlegroups.com).
 
*12.00-13.00 Lunch break*
 
**13:00 - 14:00 Summary of discussions**  

**14:00 Closing remarks**  

## Contact

For any questions please contact organisers at [dmcm-organisers@googlegroups.com](mailto:dmcm-organisers@googlegroups.com).

## Acknowledgements

We would like to thank Marion Ferraez for her help with the design of the event page.

## Organisers

### Event organisers

<p><a href="mailto:marek.ostaszewski@uni.lu">Marek Ostaszewski</a>  
<br /><a href="mailto:inna.kuperstein@curie.fr">Inna Kuperstein</a>  
<br /><a href="mailto:anna.niaraki@univ-evry.fr">Anna Niarakis</a>  
<br /><a href="mailto:a.mazein@gmail.com">Alexander Mazein</a>  
<br /><a href="mailto:reinhard.schneider@uni.lu">Reinhard Schneider</a>  
</p>

### Disease Maps Project Coordinators

<p><a href="mailto:a.mazein@gmail.com">Alexander Mazein</a>
<br /><a href="mailto:marek.ostaszewski@uni.lu">Marek Ostaszewski</a>
<br /><a href="mailto:inna.kuperstein@curie.fr">Inna Kuperstein</a>
<br /><a href="mailto:anna.niaraki@univ-evry.fr">Anna Niarakis</a>
<br /><a href="mailto:andrei.zinovyev@curie.fr">Andrei Zinovyev</a>
<br /><a href="mailto:reinhard.schneider@uni.lu">Reinhard Schneider</a>
<br /><a href="mailto:emmanuel.barillot@curie.fr ">Emmanuel Barillot</a>
<br /><a href="mailto:rudi.balling@uni.lu">Rudi Balling</a>
<br /><a href="mailto:cauffray@eisbm.org">Charles Auffray</a>
</p>

<!--## Contact-->

<!--## Co-organizers-->
